Задача 1. Даны две независимые выборки. Не соблюдается условие нормальности
x1 380,420, 290
y1 140,360,200,900
Сделайте вывод по результатам, полученным с помощью функции, имеются ли статистические различия между группами?

In [1]:
import numpy as np 
import scipy.stats as stats
x1=[380,420, 290]
y1=[140,360,200,900]
a=0.05
stats.mannwhitneyu(x1,y1)

MannwhitneyuResult(statistic=8.0, pvalue=0.6285714285714286)

pvalue=0.63 > a=0.05 ,  это означает, что статистических различий между группами не имеется.